pkg/api/v1alpha1/cloudstackmachineconfig_types.go

```diff
@@ -47,6 +47,9 @@ type CloudStackMachineConfigSpec struct {
 
 type CloudStackResourceDiskOffering struct {
 	CloudStackResourceIdentifier `json:",inline"`
+	// disk size in GB, > 0 for customized disk offering; = 0 for non-customized disk offering
+	// +optional
+	Size int64 `json:"sizeInGB"`
```

Comment:
int64 is used in cloudstack's go api lib's, so capc uses int64. Using int64 here so that it matches, otherwise, we may need to convert between each other.